src/common/constants.js

```diff
@@ -10,6 +10,7 @@ export const DISABLE_AI = false;
 export const DISPLAY_HEX_IDS = false;
 export const ENABLE_REDUX_TIME_TRAVEL = false;
 const USE_LOCAL_PARSER = false;
+const LOCAL_PARSER_PORT = 8080;
```

Comment:
I guess you should change this to match the port you're setting in `wordbots-parser`.